test/unit/app/services/service_listeners/edition_dependencies_test.rb

```diff
@@ -52,49 +52,6 @@ class ServiceListeners::EditionDependenciesTest < ActiveSupport::TestCase
         assert Whitehall.edition_services.send(service_name, subsequent_edition_of_dependable_speech).perform!
       end
     end
-
-    # given a depended-upon edition is published, then unpublished.
-    # its title is updated and it's published, causing an updated slug.
-    # we need to republish the dependent edition to reflect the updated slug.
-    # NOTE: this doesn't cover the case where a subsequent edition of the depended-upon
-    # edition changes the title/slug, leaving an outdated slug in the dependent edition.
-    test "unpublishing a depended-upon edition and #{transition}ing it again should cause dependent editions to be republished" do
-      Sidekiq::Testing.inline! do
-        disable_publishes_to_publishing_api do
-          dependable_speech, dependent_article = create_article_dependent_on_speech
-
-          expect_publishing(dependable_speech)
-          expect_republishing(dependent_article)
-          assert Whitehall.edition_services.send(service_name, dependable_speech).perform!
-
-          dependable_speech.unpublishing = create(:unpublishing, edition: dependable_speech)
-
-          # stub unpublishing worker to avoid the extra save draft calls
-          PublishingApiUnpublishingWorker.stubs(:perform_async)
-          assert Whitehall.edition_services.unpublisher(dependable_speech).perform!
-
-          dependable_speech.title = "New speech title"
-          dependable_speech.submit!
-
-          expect_publishing(dependable_speech)
-          expect_republishing(dependent_article)
-          assert Whitehall.edition_services.send(service_name, dependable_speech).perform!
-        end
-      end
-    end
-  end
```

Comment:
I'm going to investigate whether this is genuinely safe to remove. We think speeches have to depend on a news article. I think it should be safe because there shouldn't be any harm in having a removed edition preserve its dependencies, but needs confirming